config/config_test.go

```diff
@@ -125,6 +160,7 @@ func TestLoadConfig(t *testing.T) {
 						MaxExecutionTime:     Duration(time.Minute),
 						DenyHTTPS:            true,
 						NetworksOrGroups:     []string{"office", "1.2.3.0/24"},
+						Params:               "cron-job",
```

Comment:
Either remove Params from this user or add a user without params in order to verify user config without params works as expected.